reana_workflow_controller/rest/workflows.py

```diff
@@ -259,9 +256,20 @@ def get_workflows(paginate=None):  # noqa
                 if is_uuid_v4(workflow_id_or_name)
                 else query.filter(Workflow.name == workflow_id_or_name)
             )
-        if sort not in ["asc", "desc"]:
-            sort = "desc"
-        column_sorted = getattr(Workflow.created, sort)()
+        column_sorted = Workflow.created.desc()
+        if sort in ["disk-desc", "cpu-desc"]:
+            resource_type = ResourceType[sort.split("-")[0]].name
+            resource = get_default_quota_resource(resource_type)
+            query = query.join(
+                WorkflowResource,
+                and_(
+                    Workflow.id_ == WorkflowResource.workflow_id,
+                    WorkflowResource.resource_id == resource.id_,
+                ),
+            )
```

Comment:
When ordering by most cpu/disk usage, workflows that do not have entries in the `workflow_resource` table are not returned by this endpoint. This can happen when the quota updates are not done at the end of the workflow execution, but are only updated periodically by the cronjob. 

How do you think we should handle these workflows? I think it might be confusing for the user to see some workflows disappear when changing their order.

Reply:
Thanks! Very nice catch, all my workflows had quotas while I tested and I didn't notice this behavior. Fixed it to include all the workflows!

Reply:
Didn't know about `nullslast`, nice!